ios/Video/ReactVideoManager.swift

```diff
@@ -36,8 +51,8 @@ class RCTVideoManager: RCTViewManager {
         })
     }
     
-    @objc(setLicenseResultError:reactTag:)
```

Comment:
Why did you change this ? Is there an issue ?
please see: https://github.com/react-native-video/react-native-video/issues/3406